Makefile

```diff
@@ -74,6 +74,10 @@ clean:
 	rm -f oasis
 	$(GO) clean -testcache
 
+windows-build:
```

Comment:
Move this up next to `oasis`. I hope `find` works under WSL.

```suggestion
oasis.exe: $(shell find . -name "*.go" -type f) go.sum go.mod
```

Also add
```
build-windows: oasis.exe
```
 below `build`.

Reply:
Updated build-windows